src/users.py

```diff
@@ -157,6 +157,21 @@ def __iter__(self):
     def items():
         yield from var.USERS.items()
 
+def complete_match(string, users):
+    matches = []
+    string = lower(string)
```

Comment:
As an aside (and maybe I wasn't clear on that part), I'm trying to kill off `src.utilities`. That doesn't mean to kill all functions from it (although a great chunk of them will get killed off), but it does mean moving some things elsewhere. It may not be entirely possible right away, but right now none of the new interface relies on it. Let's not make our job harder by relying on it ;)